src/org/opensolaris/opengrok/web/AuthorizationFilter.java

```diff
@@ -54,13 +54,19 @@ public void doFilter(ServletRequest sr, ServletResponse sr1, FilterChain fc) thr
 
         Project p = config.getProject();
         if (p != null && !config.isAllowed(p)) {
-            LOGGER.log(Level.INFO, "access denied for uri: {0}", httpReq.getRequestURI());
+            if (httpReq.getRemoteUser() != null) {
+                LOGGER.log(Level.INFO, "Access denied for user {0} for URI: {1}",
```

Comment:
possibly enclosing the user identity in quotes would help if it has some long/strange value